{"uuid":"d39893e9-06a7-4f3a-a510-e4ba591ee532","historyId":null,"status":"broken","statusDetails":{"message":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\1-getting-started\\newTest.spec.cy.js\n\n","trace":"\n\nError: Webpack Compilation Error\n./cypress/e2e/1-getting-started/newTest.spec.cy.js\nModule not found: Error: Can't resolve '../fixtures/users.json' in 'C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\1-getting-started'\nLooked for and couldn't find the file at the following paths:\n[C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\fixtures\\users.json]\n[C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\fixtures\\users.json.js]\n[C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\fixtures\\users.json.json]\n[C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\fixtures\\users.json.jsx]\n[C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\fixtures\\users.json.mjs]\n[C:\\Users\\bharaths\\Desktop\\allure\\cypress\\e2e\\fixtures\\users.json.coffee]\n @ ./cypress/e2e/1-getting-started/newTest.spec.cy.js 3:13-46\n \n    at handle (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\@cypress\\webpack-preprocessor\\dist\\index.js:230:23)\n    at finalCallback (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:257:39)\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:304:22\n    at Compiler.emitRecords (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:499:39)\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:298:10\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:482:27\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:2818:7\n    at done (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:464:33\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\bharaths\\AppData\\Local\\Cypress\\Cache\\10.9.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (node:fs:188:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests."},"stage":"finished","attachments":[{"name":"video recording","type":"video/mp4","source":"34cb6460-ca92-469e-8863-003d6c5dc26c-attachment.mp4"}],"parameters":[],"labels":[{"name":"suite","value":"cypress/e2e/1-getting-started/newTest.spec.cy.js"}],"links":[],"start":1686838939415,"name":"Oops...we found an error preparing this test file: cypress/e2e/1-getting-started/newTest.spec.cy.js","fullName":"Oops...we found an error preparing this test file: cypress/e2e/1-getting-started/newTest.spec.cy.js","stop":1686838939415}